frontend/pages/policies/PolicyPage/components/PolicyForm/PolicyForm.tsx

```diff
@@ -423,22 +425,30 @@ const PolicyForm = ({
 
   const renderCriticalPolicy = () => {
     return (
-      <Checkbox
-        name="critical-policy"
-        className="critical-policy"
-        onChange={(value: boolean) => setLastEditedQueryCritical(value)}
-        value={lastEditedQueryCritical}
-        isLeftLabel
-      >
-        <TooltipWrapper
-          tipContent={
-            "<p>If automations are turned on, this<br/> information is included.</p>"
-          }
-          isDelayed
+      <div className="critical-checkbox-wrapper">
+        {isSandboxMode && (
+          <PremiumFeatureIconWithTooltip
+            tooltipDelayHide={500}
```

Comment:
just curious, why is the tooltip delay longer than the others? I noticed we didn't add the same delay in `NewPolicyModal`

Reply:
@ghernandez345 great eye. Per [this discussion](https://github.com/fleetdm/fleet/issues/10818#issuecomment-1494423442), this tooltip is being displayed a bit further from its trigger, so the added delay gives the user time to get their mouse over the tooltip before it disappears.